src/EnergyPlus/DXCoils.cc

```diff
@@ -229,19 +229,21 @@ void SimDXCoil(EnergyPlusData &state,
             //   call the HPWHDXCoil routine to calculate water side performance set up the DX coil info for air-side calcs
             CalcHPWHDXCoil(state, DXCoilNum, PartLoadRatio);
             //    CALL CalcDoe2DXCoil(state, DXCoilNum, CompOp, FirstHVACIteration,PartLoadRatio), perform air-side calculations
-            CalcDoe2DXCoil(state, DXCoilNum, 1, FirstHVACIteration, PartLoadRatio, FanOpMode);
+            CalcDoe2DXCoil(state, DXCoilNum, CompressorOperation::On, FirstHVACIteration, PartLoadRatio, FanOpMode);
```

Comment:
I am not sure how many more of these kinds of things we have in EnergyPlus, but replacing integers with symbols is always a good thing. Make the code much more understandable and maintainable. This is a pattern we should propagate maximally.